.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,32 @@
+name: Bug Report
```

Comment:
I don’t like these kind of issue templates tbh, I prefer the plain markdown ones where it’s easier to edit it and remove irrelevant fields if they aren’t necessary for the issue

like I don’t think we need to worry about this unless we start receiving a ton of spam issues from hacktoberfesters

Reply:
Well the current ones are terrible, so I want to fix them at least.